sdf.md

```diff
@@ -832,6 +832,15 @@ definition specify that the
 interactions offered by a Thing modeled by this sdfThing include the
 interactions modeled by the nested `sdfObject` or `sdfThing`.
 
+## Top-level Affordances and sdfData
+
+Besides their placement within an `sdfObject` or `sdfThing`, both affordances
```

Comment:
"Both affordances" and then listing three causes a double-take...
"as well as" should be enough here.

```suggestion
Besides their placement within an `sdfObject` or `sdfThing`, affordances
```